### ISSUE TITLE
2020 Census changes

### ISSUE THREAD

**Issue by AmandaDoyle** [NYCPlanning/db-pluto#246]
Changes that will need to happen to PLUTO as a result of the 2020 Census release

- [ ] Add the following fields:
- CT2020
- CB2020
- [ ] Remove the following fields:
- Tract 2010
- [ ] Update the format following fields to match the format of these values as reported in DevDB where it's text, 6 characters, with leading zeros and no decimals (e.g., 010300, 010301):
- CT2020
- CB2020 
- CT2010
- CB2010

**Comment by lseirup** [comment]
I don't think we can delete Tract2010 or change the format for CT2010 and CB2010.  Agencies may have processes in place that expect that data.

**Comment by AmandaDoyle** [comment]
Add 2 new 2020 fields with proposed formatting
Keep 2010 fields as is and phase out 2010 fields eventually

**Comment by AmandaDoyle** [comment]
@SPTKL  more detail on census fields
Leave the following fields unchanged:
- CT2010
- CB2010
- Tract2010

Add the following fields after the `CD` field (between `CD` and `CT2010`).  The formatting of these fields will be the same as the formatting of the fields in DevDB, and they'll be created the same way. 
- [BCT2020](https://github.com/NYCPlanning/db-developments/wiki/Data-Dictionary#bct2010)
- [BCTCB2020](https://github.com/NYCPlanning/db-developments/wiki/Data-Dictionary#bctcb2010)

@lseirup Can you sign off on the above?

Given that we're not changing the 2010 fields and not formatting the 2020 fields to match the 2010 fields in PLUTO the addition of the 2020 fields does not have to follow the 2010 workflow.  I think that we can do the following:
- add call to get 2020 data to [geocode.py](https://github.com/NYCPlanning/db-pluto/blob/main/pluto_build/python/geocode.py)
- add 2020 fields in [create_rpad_geo](https://github.com/NYCPlanning/db-pluto/blob/d433ed94b1ea8d7c636cb6ddf296bdf7140de7d9/pluto_build/sql/create_rpad_geo.sql)
- add 2020 fields in[ geocodes.sql](https://github.com/NYCPlanning/db-pluto/blob/d433ed94b1ea8d7c636cb6ddf296bdf7140de7d9/pluto_build/sql/geocodes.sql)
- add 2020 fields to [geocode_nones.sql](https://github.com/NYCPlanning/db-pluto/blob/d433ed94b1ea8d7c636cb6ddf296bdf7140de7d9/pluto_build/sql/geocode_nones.sql)
- Add 2020 fields and boundaries to spatialjoins.sql and spatialjoins_centroid.sql
- Add 2020 fields to create and export scripts
- Add 2020 fields to QAQC scripts

Yes, this is overly complicated.

**Comment by SPTKL** [comment]
we should also add BCT2010 and BCTCB2010 so that agencies can  have something to switch to if they no longer want CT2010, CB2010, Tract2010, and we should warn them about deprecation
